RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -100,6 +100,10 @@ boolean auto_haveMonkeyPaw()
 
 boolean auto_makeMonkeyPawWish(effect wish)
 {
+	if(get_property("_monkeyPawWishesUsed").to_int() >= 5) {
+		auto_log_info("Out of monkey paw wishes, skipping "+wish.to_string());
```

Comment:
Nice to have: use the same way to make the wish into a string here and below in the tracker. wish.to_string() vs to_string(wish)

Is there a functional difference between them?

Reply:
No difference AFAIK. Done.